lms/static/js/spec/verify_student/make_payment_step_view_ab_testing_spec.js

```diff
@@ -0,0 +1,221 @@
+                AjaxHelpers.expectPostRequest(
+                    requests, "/verify_student/create_order/", $.param( params )
+                );
+
+                // Simulate the server response
+                if ( kwargs.succeeds ) {
+                    // TODO put fixture responses in the right place
+                    AjaxHelpers.respondWithJson( requests, {payment_page_url: 'http://payment-page-url/', payment_form_data: {foo: 'bar'}} );
+                } else {
+                    AjaxHelpers.respondWithTextError( requests, 400, SERVER_ERROR_MSG);
+                }
+            };
+
+            var expectPaymentSubmitted = function( view, params ) {
+                var form;
+
+                expect(view.submitForm).toHaveBeenCalled();
+                form = view.submitForm.mostRecentCall.args[0];
+
+                expect(form.serialize()).toEqual($.param(params));
+                expect(form.attr('method')).toEqual("POST");
+                expect(form.attr('action')).toEqual('http://payment-page-url/');
+            };
+
+            var checkPaymentButtons = function( requests, buttons ) {
+                var $el = $( '.payment-button' );
+                expect($el.length).toEqual(_.size(buttons));
+                _.each(buttons, function( expectedText, expectedId ) {
+                    var buttonEl = $( '#' + expectedId),
+                        request;
+
+                    buttonEl.removeAttr('disabled');
+                    expect( buttonEl.length ).toEqual( 1 );
+                    expect( buttonEl[0] ).toHaveClass( 'payment-button' );
+                    expect( buttonEl[0] ).toHaveText( expectedText );
+                    expect( buttonEl[0] ).toHaveClass( 'action-primary-blue' );
+
+                    buttonEl[0].click();
+                    expect( buttonEl[0] ).toHaveClass( 'is-selected' );
+                    expectPaymentButtonEnabled( false );
+                    request = AjaxHelpers.currentRequest(requests);
+                    expect(request.requestBody.split('&')).toContain('processor=' + expectedId);
+                    AjaxHelpers.respondWithJson(requests, {});
+                });
+            };
+
+            beforeEach(function() {
+                window.analytics = jasmine.createSpyObj('analytics', ['track', 'page', 'trackLink']);
+
+                setFixtures( '<div id="current-step-container"></div>' );
+                TemplateHelpers.installTemplate( 'templates/verify_student/make_payment_step_ab_testing' );
+            });
+
+            it( 'A/B Testing: check Initialize method with AB testing enable ', function() {
+                var view = createView();
+                expect( view.templateName ).toEqual("make_payment_step_ab_testing");
+                expect( view.btnClass ).toEqual("action-primary-blue");
+
+            });
+
+            it( 'shows users only minimum price', function() {
+                var view = createView({}),
```

Comment:
Why is this view being passed an empty object but the view for the previous test is not being passed anything?